### PR TITLE
Configuring for pure-python-without-pypy

### PR DESCRIPTION
**Pull request by icemac**

### CODE REVIEW

**Comment by icemac** [comment]
The script does not support updating an existing PR, so the changes in .gitignore could be done manually or in another PR.

**Comment by dataflake** [comment]
That's the reason why I want to discuss that before you apply it anywhere...

File: .gitignore [hunk 1]
```diff
@@ -1,22 +1,14 @@
...
-*.dll
-*.pyc
-*.pyo
-*.so
-*.egg-info
+develop-eggs/
+dist/
+htmlcov/
+lib/
+parts/
```
Comment (dataflake) [inline .gitignore:14]: Missing:

- pyvenv.cfg
- /lib64
- *.profraw

File: tox.ini [hunk 2]
```diff
@@ -13,16 +13,28 @@ commands =
...
+[testenv:coverage]
+basepython = python3
+deps =
+    coverage
+commands =
+    coverage run -m zope.testrunner --test-path=src []
+    coverage html
+    coverage report
+
+[coverage:run]
```
Comment (dataflake) [inline tox.ini:31]: Question I asked on zopefoundation/meta#12: Does this configuration get picked up correctly when it's here and not in setup.cfg?

**Review by dataflake — CHANGES_REQUESTED** [review]

**Comment by icemac** [comment]
Replaced by #5 which includes the requested changes.

**Pull request closed without merging**